tests/unit/jobserver/test_auth_pipeline.py

```diff
@@ -84,7 +84,11 @@ def test_pipeline_with_new_user(slack_messages, strategy):
 
     social = output["social"]
     assert social.user == user
+    assert social.extra_data["id"] == "1234"
+    assert social.extra_data["login"] == "dummy-user"
+    assert "auth_time" in social.extra_data
     assert "sekret" not in social.extra_data.values()
+    assert "token_type" not in social.extra_data
```

Comment:
Nitpick: I don't think we need to test for the absence of this value or this key. Neither are added to `social.extra_data` by `pipeline`; and `pipeline` doesn't simply pass through everything it receives from GitHub's response.